docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension on Visual Studio Code
+
+You can install the {ProductShortName} extension on Visual Studio Code (VS Code) on Linux, Windows, or macOS.
+
+.Prerequisites
+
+* link:https://code.visualstudio.com[VS Code installed]
+* 8 GB RAM
+* JDK 11 or later
+* Environmental variable `JAVA_HOME` set to `jdk11`
```

Comment:
I would not call this a prerequisite because it is not something that a person is likely to have before installing the extension. This should be the first step of the installation. I'll write it for you.